CRM/Contact/BAO/Query.php

```diff
@@ -3282,6 +3288,21 @@ public function sortName(&$values) {
     }
   }
 
+  /*
+   * Where/qill clause for greeting fields
+   *
+   * @params $values
+   *
+   * @return void
+   */
+  function greetings(&$values) {
+    list($name, $op, $value, $grouping, $wildcard) = $values;
+    $name .= '_display';
+
+    $this->_qill[$grouping][] = "Greeting $op $value";
```

Comment:
How to translate this string? Perhaps `ts('Greeting %1 %2', array(1 => $op, 2 => $value))` or maybe just `$name $op $value`?